### PR TITLE
*setup script for HP Monitor

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: computer/revere-rhino-x86_64-2/setup.sh [hunk 1]
```diff
@@ -1 +1,43 @@
...
+echo Root password?
+read root_password
+
+echo Revere user password?
+read user_password
+
+echo Revere 2.4 GHz Pwd?
+read wifi_24GHz_password
+
+echo Revere 5 GHz Pwd?
```
Comment (olbender) [inline computer/revere-rhino-x86_64-2/setup.sh]: The password is the same for both, so reuse?

File: computer/revere-rhino-x86_64-2/setup.sh [hunk 2]
```diff
@@ -1 +1,43 @@
-add setup.sh content
+#!/bin/bash
+
+echo Root password?
+read root_password
+
+echo Revere user password?
+read user_password
+
+echo Revere 2.4 GHz Pwd?
```
Comment (olbender) [inline computer/revere-rhino-x86_64-2/setup.sh]: Change to "Revere wifi password?"

File: computer/revere-rhino-x86_64-2/setup.sh [hunk 3]
```diff
@@ -1 +1,43 @@
...
+   setup-available/setup-post-05-docker.sh \
+   setup-available/setup-post-06-desktop.sh \
+   .
+
+sed_arg="s/hostname=.*/hostname=revere-rhino-x86_64-2/; \
+  s/root_password=.*/root_password=${root_password}/; \
+  s/user_password=.*/user_password=( ${user_password} )/; \
+  s/lan_dev=.*/lan_dev=enp0s20f0u1u4/; \
+  s/eth_dhcp_client_dev=.*/eth_dhcp_client_dev=( enp0s20f0u1u4 )/; \
+  s%hdd=.*%hdd=${hdd}%; \
```
Comment (olbender) [inline computer/revere-rhino-x86_64-2/setup.sh]: Missing quote char

File: computer/revere-rhino-x86_64-2/setup.sh [hunk 4]
```diff
@@ -1 +1,43 @@
...
+sed -i "$sed_arg" install-conf.sh
+
+sed_arg="s/dev=.*/dev=( wlp58s0 )/; \
+  s/essid=.*/essid=( "REVERE 2.4GHz" "REVERE 5GHz" "ASTA2" )/; \
+  s/wpa2=.*/wpa2=( ${wifi_24GHz_password} ${wifi_5GHz_password} "pass3" )/; \
```
Comment (olbender) [inline computer/revere-rhino-x86_64-2/setup.sh]: Missing quote char

**Review by olbender — CHANGES_REQUESTED** [review]

File: computer/revere-rhino-x86_64-2/setup.sh (hunk 1 above)
Comment (ghost) [reply computer/revere-rhino-x86_64-2/setup.sh]: fixed

File: computer/revere-rhino-x86_64-2/setup.sh (hunk 2 above)
Comment (ghost) [reply computer/revere-rhino-x86_64-2/setup.sh]: fixed

File: computer/revere-rhino-x86_64-2/setup.sh (hunk 3 above)
Comment (ghost) [reply computer/revere-rhino-x86_64-2/setup.sh]: check fix

File: computer/revere-rhino-x86_64-2/setup.sh (hunk 4 above)
Comment (ghost) [reply computer/revere-rhino-x86_64-2/setup.sh]: check fix

File: computer/revere-rhino-x86_64-2/setup.sh (hunk 1 above)
Comment (chrberger) [reply computer/revere-rhino-x86_64-2/setup.sh]: so, the question on 5 GHz can be deleted

**Review by chrberger — CHANGES_REQUESTED** [review]

**Review by chrberger — APPROVED** [review]
LGTM

**Pull request closed without merging**